packages/components/Menu/src/MenuItem/useMenuItem.ts

```diff
@@ -53,7 +54,7 @@ export const useMenuItem = (props: MenuItemProps): MenuItemInfo => {
 
   const pressable = usePressableState({ ...rest, onPress: onInvoke });
   const itemRef = useViewCommandFocus(componentRef);
-  const keys = isSubmenu ? submenuTriggerKeys : triggerKeys;
+  const keys = disabled ? disabledTriggerKeys : isSubmenu ? submenuTriggerKeys : triggerKeys;
```

Comment:
I don't think passing `['ArrowLeft', 'ArrowRight']` key triggers to disabled menu items is necessary. I think both keys are handled natively on win32 so focus will move to the next item if an arrow key is pressed. So I'd pass in nothing if the menu item is disabled.